multiqc/utils/search_patterns.yaml

```diff
@@ -203,6 +203,8 @@ star:
     fn: '*Log.final.out'
 star_genecounts:
     fn: '*ReadsPerGene.out.tab'
+sortmerna:
+    contents: 'Command: sortmerna'
```

Comment:
That's an excellent point, I hadn't thought about filenames. Ok then yes, one of those rarer terms would be ideal.